src/pages/STG_SettingsManager.page

```diff
@@ -136,6 +136,8 @@
                                 <li class="slds-tree__item" role="treeitem" aria-level="2" id="idPanelRelNav"><a href="#" onclick="ShowPanel('idPanelRel');return false;" class="slds-grow slds-m-left_large">{!$Label.stgNavRelationships}</a></li>
                                 <li class="slds-tree__item" role="treeitem" aria-level="2" id="idPanelRelReciprocalNav"><a href="#" onclick="ShowPanel('idPanelRelReciprocal');return false;" class="slds-grow slds-m-left_large">{!$Label.stgNavRelReciprocal}</a></li>
                                 <li class="slds-tree__item" role="treeitem" aria-level="2" id="idPanelRelAutoNav"><a href="#" onclick="ShowPanel('idPanelRelAuto');return false;" class="slds-grow slds-m-left_large">{!$Label.stgNavRelAutoCreate}</a></li>
+                                <!-- HELP TEXT MAPPING -->
```

Comment:
If we can figure out where we want to slot this in the settings now, maybe let's see if we can figure that out